tests/functional/adapter/store_test_failures_tests/test_store_test_failures.py

```diff
@@ -1,9 +1,22 @@
+import pytest
 from dbt.tests.adapter.store_test_failures_tests.test_store_test_failures import (
     StoreTestFailuresBase,
 )
 
 
+TEST_AUDIT_SCHEMA_SUFFIX = "dbt_test__aud"
+
+
 class TestBigQueryStoreTestFailures(StoreTestFailuresBase):
+    @pytest.fixture(autouse=True)
+    def teardown_method(self, project):
```

Comment:
should this be something we go change in all tests that have a `clean_up` method instaed

Reply:
I don't think we need to actively change it, but if we're adding new items or updating something, then I'd change it as part of that. Also, you still need `scope="function"`, so it would be `@pytest.fixture(scope="function", autouse=True)`